src/Files.App/Actions/Content/PreviewPopup/LaunchPreviewPopupAction.cs

```diff
@@ -3,28 +3,33 @@
 
 using Files.App.Commands;
 using Files.App.Contexts;
-using Files.Backend.Services;
 
 namespace Files.App.Actions
 {
 	internal class LaunchPreviewPopupAction : ObservableObject, IAction
 	{
-		private readonly IContentPageContext context = Ioc.Default.GetRequiredService<IContentPageContext>();
+		private readonly IContentPageContext context;
+
```

Comment:
You forgot to assign it in the constructor

Reply:
Fixed